wake-ec2.yml

```diff
@@ -1,12 +1,14 @@
 ---
-
-# Change state of all instances in the EC2 environment
   - hosts: localhost
     tasks:
-      - name: Deploy ec2.py
-        file: src=ec2.py dest=./ec2.py mode=0744
+      - name: Copy ec2.py to /tmp
+        copy: src="./ec2.py" dest="/tmp/ec2.py" mode=0744
+      - name: Copy ec2.ini to /tmp
+        copy: src="./ec2.ini" dest="/tmp/ec2.ini" mode=0644
       - name: Get stopped instances
         shell: "./ec2.py --stopped --refresh-cache"
+        args:
+          chdir: /tmp
```

Comment:
I initially thought about it, but main reason I avoided that is to not slow things down anymore. On further tests, instances enter 'pending' state first, so they would not be listed anyway (we filter for only 'running' instances in ec2.ini). Instead, let's add option to invalidate cache to ec2.py.